dev-python/scikit-optimize/scikit-optimize-0.8.1.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..9} )
+inherit distutils-r1
+
+DESCRIPTION="Sequential model-based optimization library"
+HOMEPAGE="https://scikit-optimize.github.io/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
```

Comment:
Is this intended?  Have you tested the ebuild on ppc64 and arm64?

Reply:
Oops, this is a leftover from the previous ebuild.
Sorry about this.
fixed.